### PR TITLE
Added @objc annotation style

### PR DESCRIPTION
**Pull request by dfreniche**
Discussed here: https://digitalcrew.teamwork.com/#/tasks/14241972

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -1476,6 +1477,27 @@ The compiler will show a warning ---> 'titleLabel' is deprecated: This has to be
...
+## Objc annotation
+
+Sometimes we need to add an `@objc` annotation to a `var`, `let` or `func`. In that case we'll leave the `@objc` annotation in its own line, add a line break and then the statement it's annotating in a new line.
+
+**Preferred**:
+
+```swift
+@objc 
+func refreshAllProjects(deleteAllData: Bool = true) {
+        retrieveProjects()
```
Comment (chillok) [inline README.markdown:1490]: whitespace :P

File: README.markdown [hunk 2]
```diff
@@ -1476,6 +1477,27 @@ The compiler will show a warning ---> 'titleLabel' is deprecated: This has to be
...
+}
+```
+
+**Not Preferred**:
+
+```swift
+@objc func refreshAllProjects(deleteAllData: Bool = true) {
+        retrieveProjects()
```
Comment (chillok) [inline README.markdown:1499]: whitespace :P

File: README.markdown (hunk 1 above)
Comment (dfreniche) [reply README.markdown:1490]: Fixed in 9f1e2ef8163b303f4813d5d3c8eb866c05a44fe0

File: README.markdown (hunk 2 above)
Comment (dfreniche) [reply README.markdown:1499]: Fixed in 9f1e2ef8163b303f4813d5d3c8eb866c05a44fe0

**Review by denisq1991 — APPROVED** [review]

**Review by rkreutz-teamwork — APPROVED** [review]

**Pull request merged**